wiki/Beatmaps/Marathon/en.md

```diff
@@ -0,0 +1,15 @@
+# Marathon
+
+**Marathon** [beatmaps](/wiki/Beatmaps) are beatmaps which are 5 minutes or longer. They are often collaborative efforts, as mapping longer songs can be tiring when done alone.
+
+## History
+
+When marathon beatmaps were first introduced they had to be at least 6 minutes long and could only include a single difficulty (with one exception; each beatmapset could include a single [osu!taiko](/wiki/Game_Modes/osu!taiko) difficulty as well).
+
+They also had to be named "Marathon" to bypass the Beatmap Submission System at the time and prevent the beatmapset from entering the Pending category. Upon ranking, marathon beatmaps would enter the Approved category instead of the Ranked category to signal that these were marathon maps. This category was later abandoned in favor of the Ranked category.
```

Comment:
```suggestion
Marathon beatmaps also had to have their single difficulty be named "Marathon" to bypass the Beatmap Submission System at the time and prevent the beatmapset from entering the [Pending](/wiki/Beatmaps#Pending) category. Upon ranking, marathon beatmaps would enter the Approved category instead of the [Ranked](/wiki/Beatmaps#ranked) category to signal that these were marathon maps. The category was later abandoned in favor of the Ranked category.
```